Lombiq.VueJs/Assets/Scripts/helpers/rollup-pipeline.js

```diff
@@ -1,72 +1,111 @@
-const all = require('gulp-all');
+const fs = require('fs');
 const path = require('path');
-const gulp = require('gulp');
-const plumber = require('gulp-plumber');
-const rollup = require('gulp-rollup');
-const rename = require('gulp-rename');
-const uglify = require('gulp-uglify');
-const log = require('fancy-log');
+const { minify } = require('terser');
+const { rollup } = require('rollup');
+
+const { handleErrorObject } = require('nodejs-extensions/scripts/handle-error');
+
+function createDirectory(directoryPath) {
+    return fs.existsSync(directoryPath) ? Promise.resolve() : fs.promises.mkdir(directoryPath);
+}
+
+function handleRollupError(error) {
+    const stack = error?.stack?.split(/[\n\r]+/).map((line) => line.trim()) ?? [];
+    const extensionRegex = /\.(js|vue):(\d+)(:(\d+))?.*/;
```

Comment:
Let's use [named groups](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Groups_and_Backreferences#using_named_groups) for more clarity down the line (code, age):
```suggestion
    const extensionRegex = /\.(js|vue):(?<line>\d+)(:(?<column>\d+))?.*/;
```